docs/README.md

```diff
@@ -0,0 +1,14 @@
+# 🧩 Development of [ReVanced patches](https://github.com/revanced/revanced-patches)
```

Comment:
```suggestion
# 🧩 Development of Patches
```

Reply:
A link here is good if you wanna jump back to the repository.

Reply:
We can keep the link then.
```suggestion
# 🧩 Development of [Patches](https://github.com/revanced/revanced-patches)
```

Reply:
Since we link to a repository by ReVanced, for consistency we keep the name ReVanced in there too. You should see that all links to our repos are "ReVanced + Name".